pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/GrpcSendingMailbox.java

```diff
@@ -147,4 +150,9 @@ private MailboxContent toMailboxContent(TransferableBlock block)
       _statMap.merge(MailboxSendOperator.StatKey.SERIALIZATION_TIME_MS, System.currentTimeMillis() - start);
     }
   }
+
+  @Override
+  public String toString() {
+    return "g" + _id;
```

Comment:
I've added some toString to mailboxes to make it easier to understand them in debug/trace logs